theme.js

```diff
@@ -0,0 +1,43 @@
+import { slug } from 'github-slugger'
+import path from 'node:path'
+import {
+  MarkdownTheme,
+  MarkdownThemeRenderContext
+} from 'typedoc-plugin-markdown'
+
+const externalLinkRegex = /^(http|ftp)s?:\/\//
+
+export function load(app) {
+  app.renderer.defineTheme('custom-markdown-theme', CustomMarkdownTheme)
+}
+
+export class CustomMarkdownTheme extends MarkdownTheme {
+  getRenderContext(pageEvent) {
+    return new CustomMarkdownThemeContext(pageEvent, this.application.options)
+  }
+}
+
+class CustomMarkdownThemeContext extends MarkdownThemeRenderContext {
+  relativeURL = url => {
```

Comment:
Why here we have `method = () => {}` style and above `method () {}` style?

If there is no reason, let’s normalize.

Reply:
TIL that if the method in the parent class is declared via the arrow function, it can only be overridden by the arrow function. 

![image](https://github.com/evilmartians/astro-typedoc/assets/35267898/81cbdcc3-7823-4f8b-a3b9-5cc882b03f82)